### PR TITLE
Add Makefile to compile js/py files

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+VIM_COMPILER:=vim -N --cmd 'set rtp+=$$PWD' -c q -u
```
Comment (mattn) [inline Makefile]: Can this be `.` instead of `$PWD`? (for windows)

Comment (blueyed) [inline Makefile]: Sure.  But we could also use make's `$(CURDIR)`?!
Just `.` seems nice and simple though.

**Comment by mattn** [comment]
LGTM. Could you please add `make` into here?

https://github.com/vim-jp/vim-vimlparser/blob/a26f34a434307860552f6f9f1df2b4ffd2a7ab89/.travis.yml#L29

**Comment by blueyed** [comment]
Should it check that it does not generate new compiled files then, i.e. check/test that the compiled files are up to date?

**Comment by mattn** [comment]
Yes, just checking compilation errors for now. 👍

**Comment by blueyed** [comment]
Done.  Should fail then now.

**Comment by blueyed** [comment]
> just checking compilation errors for now

What kind of errors do you mean? Just errors during compilation?

We could also run `python` on the `.py` one to detect reserved keywords as in https://github.com/vim-jp/vim-vimlparser/pull/43#issuecomment-304866269, but that might be too much for the scope of this PR.

**Comment by mattn** [comment]
Yes, I'm not going to check output of the compilation. Just checking build. Thank you.

**Pull request merged**